### PR TITLE
feat(constants): update deploy parameters

### PR DESCRIPTION
**Pull request by PierrickGT**

### CODE REVIEW

File: src/Constants.sol [hunk 1]
```diff
@@ -122,16 +126,16 @@ abstract contract Constants {
   string internal constant USDC_SYMBOL = "USDC";
   string internal constant WETH_SYMBOL = "WETH";
 
-  string internal constant PT_USDC_NAME = "PoolTogether Prize USD Coin";
-  string internal constant PT_USDC_SYMBOL = "PTUSDC";
+  string internal constant PT_USDC_NAME = "Prize USDC - Aave";
```
Comment (asselstine) [inline src/Constants.sol]: Prize USDC.e - Aave

File: src/Constants.sol [hunk 2]
```diff
@@ -122,16 +126,16 @@ abstract contract Constants {
...
+  string internal constant PT_USDC_SYMBOL = "pUSDC";
```
Comment (asselstine) [inline src/Constants.sol]: pUSDC.e

File: script/helpers/generateContractList.ts [hunk 3]
```diff
@@ -256,7 +256,7 @@ export const getVaultInfos = (transactionData: any) => {
   }
 
   return { underlyingAssetAddress, name, symbol };
-}
+};
 
 export const generateVaultList = (vaultDeploymentPath: string): VaultList => {
```
Comment (asselstine) [inline script/helpers/generateContractList.ts:262]: Remember to update token names + symbols in this file

**Review by asselstine — CHANGES_REQUESTED** [review]

**Comment by asselstine** [comment]
One other thing:

RNG auction initial target: 0.24
Relay auction initial target: 0.132

File: src/Constants.sol (hunk 1 above)
Comment (PierrickGT) [reply src/Constants.sol]: Fixed in this commit: https://github.com/GenerationSoftware/pt-v5-mainnet/pull/3/commits/dfcecebb78e3cace69813181beb20098400cb326

File: src/Constants.sol (hunk 2 above)
Comment (PierrickGT) [reply src/Constants.sol]: Fixed in this commit: https://github.com/GenerationSoftware/pt-v5-mainnet/pull/3/commits/dfcecebb78e3cace69813181beb20098400cb326

**Review by trmid — APPROVED** [review]

File: src/Constants.sol [hunk 4]
```diff
@@ -75,26 +78,29 @@ abstract contract Constants {
...
-  function _getFirstDrawStartsAt() internal pure returns (uint48) {
-    return uint48(1693594800); // Sep 1, 2023, 7:00:00 PM UTC
+  function _getFirstDrawOpensAt() internal pure returns (uint48) {
+    return uint48(1697767200); // Oct 20, 2023, 2:00:00 AM
   }
 
   // RngAuctions
   uint64 internal constant AUCTION_DURATION = 6 hours;
   uint64 internal constant AUCTION_TARGET_SALE_TIME = 1 hours;
   uint256 internal constant AUCTION_MAX_REWARD = 10000e18;
```
Comment (asselstine) [inline src/Constants.sol]: 500e18

Comment (PierrickGT) [reply src/Constants.sol]: Done: https://github.com/GenerationSoftware/pt-v5-mainnet/pull/3/commits/5752f77bbbfd070ba962bf730fb347e89b5652b4

**Review by asselstine — APPROVED** [review]

**Review by trmid — APPROVED** [review]

**Pull request merged**